### PR TITLE
Checksum b64 for bindings

### PR DESCRIPTION
**Pull request by dbnicholson**
Provide checksum_b64 functions that can be used from GI. This allows external GI programs to use ostree's modified base64 encoding without having to reimplement it.

### CODE REVIEW

File: src/libostree/ostree-core.c [hunk 1]
```diff
@@ -1273,6 +1273,20 @@ ostree_checksum_to_bytes_v (const char *checksum)
...
 /**
+ * ostree_checksum_b64_to_bytes:
+ * @checksum: An ASCII checksum
+ *
+ * Returns: (transfer full) (array fixed-size=32): Binary version of @checksum.
+ */
+guchar *
+ostree_checksum_b64_to_bytes (const char *checksum)
+{
+  guchar *ret = g_malloc (44);
```
Comment (cgwalters) [inline src/libostree/ostree-core.c]: This only needs to be 32, right?

**Comment by cgwalters** [comment]
bot, add author to whitelist

File: src/libostree/ostree-core.c (hunk 1 above)
Comment (dbnicholson) [inline src/libostree/ostree-core.c]: Duh. I fixed the annotation but didn't change the allocation.

**Comment by dbnicholson** [comment]
Ugh. Accidentally deleted my branch and github closed this. Continue on #398, please.

**Pull request closed without merging**